project/project-grader.json

```diff
@@ -3,7 +3,7 @@
   "code": "ready!set!responsiveimages!",
   "tests": [
     {
-      "description": "&lt;img&gt;s have max-widths of 100%",
+      "description": "<img> have max-widths of 100%",
```

Comment:
Yeah, it was advised by AMO not to use HTML inside the descriptions. Using the property `Node.textContent`, doesn't allow HTML.

I also planned to add basic markdown support. Bold, italic and code would be supported.